app/res/layout/pin_auth_view.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/auth_prompt_text"
+        android:layout_centerHorizontal="true"
+        android:layout_margin="@dimen/standard_spacer_large"
+        android:textSize="@dimen/text_large" />
+
+    <EditText
```

Comment:
You might be able to make this similarity easier by separating out the common elements here into their own "component" layout and using an <include> to bring them in. Will keep things the same without replicating code. screen_login.xml does that with the top banner if you want an example.